source/js/menu.js

```diff
@@ -0,0 +1,10 @@
+let navButton = document.querySelector('.page-header__toggle');
+let navMain = document.querySelector('.main-nav');
```

Comment:
если значения переменных не меняются, используй всегда const вместо let
```suggestion
const navButton = document.querySelector('.page-header__toggle');
const navMain = document.querySelector('.main-nav');
```